api/src/domain/tags/add.js

```diff
@@ -0,0 +1,19 @@
+import { get as getVersion } from '../configurations/version';
+import tagsQueries from '../../queries/tags';
+import validateTag from './validator';
+
+export default async (projectId, environmentName, configName, selector, rawName) => {
+    const name = rawName.toLowerCase();
```

Comment:
Can you check this in `validateTag` and throw an error if it's not all lower case ?